spectron/core/layout/settings-menu/settings-menu.func.ts

```diff
@@ -0,0 +1,39 @@
+import * as spectron from "spectron";
+import {boot, shutdown} from "../../../util/util";
+
+const assert = require("assert");
+
+
+describe("settings menu", function () {
+
+    let app: spectron.Application;
+
+    it("displays name of the active user", async function () {
+
+        const user = {
+            "id": "api_demo",
```

Comment:
Not sure if this matters for anywhere in this test, but the platform prefix doesn't match the url.